src/code/code_800FCE80.c

```diff
@@ -132,6 +139,9 @@ f32 Math_FAtanF(f32 x) {
     }
 }
 
+/**
+ * @return angle from vector (1,0) to (x,y) in radians, in (-pi,pi] range
```

Comment:
Thoughts on maybe
`@return angle between (x,y) and the x-axis in radians, in (pi,pi] range`?

Reply:
I'm not convinced idk
The orientation ((1,0) to (x,y)) is less clear and "x axis" isn't explicitly oriented

But suggestions on phrasing this are welcome, mine fits how I look at it but there are probably people out there who wouldn't get what this means

Reply:
Perhaps just swapping `to (x,y)` and `from vector (1,0)`, like 
`@return angle to (x,y) from vector (1,0) in radians, in (-pi,pi] range`
cause at the moment, when I first read it, I imagine it is describing the vector that starts at `(1,0)` and points to `(x,y`) which I'm pretty sure is not what you want. This would clear that up as really only `(1,0)` describes the vector.

Reply:
Maybe adding "around (0,0)" somewhere could help with that?

Reply:
Possibly, I would need to see an example. (tbh this isn't too big a deal and might just be a me problem).

Reply:
anons suggestion of `@return angle to (x,y) from vector (1,0) in radians, in (-pi,pi] range`
makes it easier for me to follow along, at least

Reply:
```suggestion
 * @return angle to (x,y) around (0,0) from vector (1,0) in radians, in (-pi,pi] range
```
```suggestion
 * @return angle to (x,y) from vector (1,0) around (0,0) in radians, in (-pi,pi] range
```

Reply:
I like the second one better :)

Reply:
yea thats fine with me as well